content/docs/command-reference/plots/show.md

```diff
@@ -1,6 +1,7 @@
 # plots show
 
-Generate a plot image from from a [metrics](/doc/command-reference/plots) file.
+Generate a plot image from from a
+[continuous metrics](/doc/command-reference/plots) file.
```

Comment:
may be we can use "series"? (like in one of the descriptions above?)

Reply:
I also vote for series and/or data series here.